examples/bin/dsql-main

```diff
@@ -57,15 +58,15 @@ def do_query(url, sql, context, timeout, user, ignore_ssl_verification, ca_file,
       query_context = context
     elif int(context.get('timeout', 0)) / 1000 < timeout:
```

Comment:
The shebang line specifies `python`, which typically means "python 2" and is consistent with the invocation of the script in `examples/bin/dsql`:
```bash
  exec python2 "$WHEREAMI/dsql-main" "$@"
```

If we want to keep the script content as python2 (except for the python3 `print()` function), then this line can be changed to:
```python
  elif int(context.get('timeout', 0)) / 1000.0 < timeout:
```
instead of using
```python
from __future__ import division
```
which minimizes the (potentially confusing) combination of python2 and python3 in the same file.

Reply:
Yep, I agree with you. Thanks a lot.